packages/shared/src/focusable.ts

```diff
@@ -135,12 +135,14 @@ export class Focusable extends FocusVisiblePolyfillMixin(SpectrumElement) {
     protected manageAutoFocus(): void {
         if (this.autofocus) {
             /* Trick :focus-visible polyfill into thinking keyboard based focus */
-            this.dispatchEvent(
-                new KeyboardEvent('keydown', {
-                    code: 'Tab',
-                })
-            );
-            this.focusElement.focus();
+            requestAnimationFrame(() => {
+                this.dispatchEvent(
+                    new KeyboardEvent('keydown', {
+                        code: 'Tab',
+                    })
+                );
+                this.focusElement.focus();
+            });
```

Comment:
Wait a frame so that you can know that the polyfill has been applied.

Reply:
I find it's better to be explicit about what your waiting for in cases like this (e.g. say for some reason you need to chain an extra animation frame for the polyfill because of something else -- that'd break this code).

At the very least I'd add a comment

Reply:
Great point! Found a way not to have the polyfill reach into the library code like this. Let me know what you think.